### ISSUE TITLE
[CLI] Save files with wandb.save without the folder structure

### ISSUE THREAD

**Issue by konstantinjdobler** [wandb/client#2242]
**Description**
When I call e.g.  `wandb.save(config.results_dir + "latest_checkpoint.pth")` the entire folder structure will be saved. This can be annoying if I want to restore files or inspect them online in some cases. Is there a way to only store the file without the folder structure in wandb?

**Comment by tyomhak** [comment]
Hey @konstantinjdobler , as far as I'm aware it's not possible yet. Thank you for the feature request :)

**Comment by konstantinjdobler** [comment]
Hey @tyomhak, I have indeed just found a way to do this after inspecting the wandb source code. The following snippet does what I want:
```python
import os
filepath = config.results_dir + "latest_checkpoint.pth"
wandb.save(filepath, base_path=os.path.dirname(filepath))
```
It's easy enough to do once you know it, but I had to go digging to find it and it is not really intuitive. I can open a PR if you want to turn it into an additional argument for `wandb.save`, otherwise this should probably be added to the documentation of `wandb.save` as a hint.
